sync-orders-to-bigQuery/README.md

```diff
@@ -0,0 +1,37 @@
+# Moltin -> BigQuery
+
+> A simple serverless integration to help you send Moltin order data (including order items) to Google BigQuery.
+
+📚 [Moltin API reference](https://docs.moltin.com/) &mdash; 📚 [BigQuery API reference](https://cloud.google.com/bigquery/docs/reference/rest/)
+
+## 🛠 Pre-requisites
+1. A Google project with a dataset and two tables created. One table will receive orders and the other, order items. Your orders table should have a schema matching the field names in the `filterOrderJson` function in `mapping.js`, your order items table schema should match the field names in the `filterItemJson` in the same mapping file.
+
+2. A Google KMS keyring and key.
+
+3. Serverless CLI installed (`npm install -g serverless`)
+
+## 🛠 Setup
+
+1. Clone the repository.
+2. Encrypt your Moltin Client Secret using your Google keyring and key.
+3. Change directory into the repository (`cd integration-examples/sync-moltin-bigQuery`) and run `yarn`.
```

Comment:
It would be great to follow the other READMEs and structure this as a step.

### 1. Download the example

Clone the repository:
```bash
git clone git@github.com:moltin/integration-examples.git
```

Install dependencies with Yarn
```bash
cd integration-examples/sync-moltin-big-query
yarn
```

Reply:
Try now :)